rediscache/__init__.py

```diff
@@ -147,8 +150,16 @@ def refreshvalueinthread(key):
                     return direct_value
 
                 # Lets create a key from the function's name and its parameters values
-                values = ",".join([value.__str__() for value in args])
-                dict_values = ",".join([str(key) + "='" + value.__str__() + "'" for key, value in kwargs.items()])
+                if include_args:
+                    values = ",".join([value.__str__() for value_id, value in enumerate(args) if value_id in include_args])
+                else:
+                    values = ",".join([value.__str__() for value in args])
+
+                if include_kwargs:
+                    dict_values = ",".join([str(key) + "='" + value.__str__() + "'" for key, value in kwargs.items() if key in include_kwargs])
+                else:
+                    dict_values = ",".join([str(key) + "='" + value.__str__() + "'" for key, value in kwargs.items()])
```

Comment:
A dedicated function to convert the parameters into a key might be useful to decrease the complexity of `wrapper()` and should make it easy to unit test. We need them! ;-)